gobblin-utility/src/main/java/org/apache/gobblin/util/PullFileLoader.java

```diff
@@ -260,6 +263,8 @@ private Config loadJavaPropsWithFallback(Path propertiesPath, Config fallback) t
     PropertiesConfiguration propertiesConfiguration = new PropertiesConfiguration();
     try (InputStreamReader inputStreamReader = new InputStreamReader(this.fs.open(propertiesPath),
         Charsets.UTF_8)) {
+      propertiesConfiguration.setDelimiterParsingDisabled(ConfigUtils.getBoolean(fallback,
```

Comment:
This disables the parsing for all configuration keys. What about a pull file has some key that needs parsing and another key that doesn't need?

Reply:
I don't see an API in Apache Commons to make granulation calls on delimitation. Do you think we should address such use case?